internal/cli/apis.go

```diff
@@ -324,6 +326,52 @@ auth0 apis delete <id>`,
 	return cmd
 }
 
+func openApiCmd(cli *cli) *cobra.Command {
+	var inputs struct {
+		ID string
+	}
+
+	cmd := &cobra.Command{
+		Use:     "open",
+		Args:    cobra.MaximumNArgs(1),
+		Short:   "Open API settings page in Auth0 Manage",
+		Long:    "Open API settings page in Auth0 Manage.",
+		Example: "auth0 apis open <id>",
+		PreRun: func(cmd *cobra.Command, args []string) {
+			prepareInteractivity(cmd)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) == 0 {
+				err := apiID.Pick(cmd, &inputs.ID, cli.apiPickerOptions)
+				if err != nil {
+					return err
+				}
+			} else {
+				inputs.ID = args[0]
+			}
+
+			// Heuristics to determine if this a valid ID, or an audience value
+			if _, err := url.ParseRequestURI(inputs.ID); err == nil || len(inputs.ID) != 24 {
+				if err := ansi.Waiting(func() error {
+					api, err := cli.api.ResourceServer.Read(url.PathEscape(inputs.ID))
+					if err != nil {
+						return err
+					}
+					inputs.ID = auth0.StringValue(api.ID)
+					return nil
+				}); err != nil {
+					return fmt.Errorf("An unexpected error occurred while trying to get the API Id for '%s': %w", inputs.ID, err)
```

Comment:
NAB: I know we're doing this in other places already, but I'd suggest starting error messages with lowercase (as well as avoiding ending punctuation). More context here: https://github.com/golang/go/wiki/CodeReviewComments#error-strings